src/components/EncounterDetails/EncounterDetails.css

```diff
@@ -0,0 +1,101 @@
+details, .damage-mods, .senses, .main-details, .foe-stats, .foe-physical, .foe-speeds, .base-box, .party-stats, .foe-proficiencies, .proficiencies {
+  display: flex;
+  box-sizing: border-box;
+}
+
+.main-details {
+  max-width: 90vw;
+  min-width: 85vw;
+  align-items: flex-start;
+  align-content: center;
+  align-self: center;
+}
+
+.party-stats {
+  justify-content: center;
+}
+
+.base-box {
+  background-color: #131316;
+  color: #4AA3FC;
+  margin-top: 1em;
+  max-width: 90vw;
+  align-self: flex-start;
+  min-width: 85vw;
+  border: 1px solid #BCBEF0;
+  flex-direction: column;
+}
+
+summary {
+  color: #3D157A;
+  font-size: large;
+  font-weight: 600;
+  background-color: #EF81E5;
+  padding-left: 10px;
+}
+
+p {
+  background-color: #BCBEF0;
+  color: #3D157A;
+  margin: 1em;
+  min-width: 2em;
+  min-height: 1.25em;
+  padding: 0.25em;
+}
+
+h3 {
+  align-self: flex-start;
+  margin: .5em;
+}
+
+details, p {
+  text-align: left;
+}
+
+details {
+  background-color: #EF81E5;
```

Comment:
As discussed in slack, let's change the bulk of this pink to a darker shade of magenta. #901386 is the shad that I found in the coolors picker that we liked. And here's a screenshot for reference.
![Screenshot 2024-02-07 at 6 24 07 PM](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/133910120/d8c393e4-d750-4d82-9f52-215693127c39)

Reply:
Updated, will push that up!